spec/models/organization_spec.rb

```diff
@@ -71,7 +71,9 @@
 
     it "downcases slug" do
       organization.slug = "HaHaHa"
-      organization.save
+      perform_enqueued_jobs do
```

Comment:
Downcasing `slug` is not related to the async jobs, so there's no need to run them while testing it.
I suggest adding another test that checks that the cache busting job was enqueued if you want to check it.